src/www/containers/userprofile/UserProfile.scss

```
@@ -0,0 +1,405 @@
+          padding: 0.5rem;
+          border: none;
+          cursor: pointer;
+          border-radius: 5px;
+
+          > img {
+            height: 1rem;
+          }
+        }
+      }
+    }
+
+    .Username {
+      font-size: 1.5rem;
+    }
+
+    .Info {
+      font-size: 1rem;
+      margin-top: -0.5rem;
+      max-width: 15rem;
+      overflow-wrap: break-word;
+
+      &-edit {
+        .Edit-bio-input {
+          resize: none;
+          height: 5rem;
+          border-radius: 0.3rem;
+          width: 100%;
+        }
+
+        .Save-button,
+        .Cancel-button {
+          background-color: rgba($color: white, $alpha: 0.3);
+          border: 1px solid rgba($color: black, $alpha: 0.3);
+          border-radius: 0.3rem;
+          box-shadow: 0.5px 0.5px 0.5px 0.5px black;
+          cursor: pointer;
+          font-size: 1rem;
+          margin-top: 0.5rem;
+          margin-right: 0.5rem;
+          outline: none;
+          padding: 0.2rem;
+          transition: 0.1s ease-in-out;
+          width: 5rem;
+
+          &:hover {
+            background-color: rgba($color: rgb(192, 180, 180), $alpha: 0.3);
+          }
+        }
+      }
+    }
+
+    .Edit-profile-button {
+      &-off {
+        background-color: rgba($color: white, $alpha: 0.3);
+        border: 1px solid rgba($color: black, $alpha: 0.3);
+        border-radius: 0.3rem;
+        box-shadow: 0.5px 0.5px 0.5px 0.5px black;
+        cursor: pointer;
+        font-size: 1.2rem;
+        margin-top: 1rem;
+        outline: none;
+        padding: 0.5rem;
+        transition: 0.1s ease-in-out;
+        width: 15rem;
+
+        &:hover {
+          background-color: rgba($color: rgb(192, 180, 180), $alpha: 0.3);
+        }
+      }
+
+      &-on {
+        display: none;
+      }
+    }
+  }
+
+  .Panel {
+    margin-left: 5rem;
+    width: 100%;
+
+    .List {
+      border-bottom: 1px solid black;
+      display: flex;
+      flex-direction: row;
+      list-style-type: none;
+      margin: 0;
+      padding: 0;
+
+      > li {
+        cursor: pointer;
+        padding: 0.5rem;
+        padding-top: 0;
+
+        &:hover {
+          border-bottom: 2px solid orangered;
+        }
+
+        &:not(:first-of-type) {
+          margin-left: 4rem;
+        }
+
+        &.Active-tab {
+          border-bottom: 2px solid red;
+        }
+      }
+    }
+
+    .Content {
+      margin-top: 1rem;
+
+      > :not(.active) {
+        display: none;
+      }
+
+      .Actions,
+      .Teams {
+        > div {
+          background-color: rgb(209, 206, 218);
+          border: 1px solid rgba($color: black, $alpha: 0.4);
+          border-radius: 0.5rem;
+          box-shadow: 0px 1px black;
+
+          &:not(:first-of-type) {
+            margin-top: 1rem;
+          }
+
+          > h3 {
+            margin-left: 1rem;
+            text-align: left;
+          }
+
+          > ul {
+            list-style-type: none;
+            text-align: left;
```

Remove list's default left padding.
```suggestion
            padding: 0;
            text-align: left;
```